examples/fastly/README.md

```diff
@@ -0,0 +1,16 @@
+# F1 Stats on Fastly Compute@Edge
+
+This is an example data source to retrieve our dataset using Fastly Compute@Edge.
+
+## Notable things about Fastly Compute@Edge
+
+- Fastly Compute@Edge runs WebAssembly, which means any language that can compile to WebAssembly could be used.
+- We are using their [JavaScript SDK](https://github.com/fastly/js-compute-runtime/) which is based on SpiderMonkey (the JavaScript engine used in Firefox).
+- We are using a Fastly Edge Dictionary to store the PlanetScale database credentials.
+
+## Setup
+
+1. Run `npm install`
+2. Run `npm run dev` to start the app locally.
```

Comment:
Trying this out now. `npm run dev` isn't in the `package.json`, could you add that?

Reply:
yes it should be 'fastly compute serve'

Reply:
added in [`9eb3974` (#19)](https://github.com/planetscale/f1-championship-stats/pull/19/commits/9eb39748239a0992f091644c7b7a559c30852d01)